package.json

```diff
@@ -35,7 +35,7 @@
     "mkdirp": "^0.5.1",
     "pify": "^3.0.0",
     "tmp": "0.0.29",
-    "uglify-js": "^2.6.2",
+    "uglify-js": "^3.0.21",
```

Comment:
Could you make `uglify-js` a peerDependency so we can use custom versions? Also, it might be better to use `uglify-es`, since it's a superset of `uglify-js` with additional support for ES6? Thanks!